documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,353 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="new-migrating-virtual-machines-cli_{context}"]
+
+ifdef::vmware[]
+= Migrating a VMware vSphere source provider
+
+Before you can migrate a VMware vSphere source provider using the CLI, you must obtain the SHA-1 fingerprint of a vCenter host, as described in the procedure that follows.
+
```

Comment:
@anarnold97 Instead of "as described in the procedure that follows," I want to insert an xref to  the file obtaining-vmware-fingerprint.adoc and "xref:obtaining-vmware-fingerprint_{context}[vCenter SHA-1 fingerprint]" does not work as it does elsewhere. 

Note that this is in an ifdef statement with cointext=vmware.